### ISSUE TITLE
Make the viewport responsive ready

### ISSUE THREAD

**Issue by miguelmich** [cuartas15/proyecto#7]
# What
Let's adjust the viewport to make it responsive:

```html
<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

**Comment by cuartas15** [comment]
I knew I was forgetting something there.

**Comment by miguelmich** [comment]
Fixed on 870f87a
